Adyen/Model/LegalEntityManagement/AULocalAccountIdentification.cs

```diff
@@ -0,0 +1,208 @@
+        /// </summary>
+        /// <value>The 6-digit [Bank State Branch (BSB) code](https://en.wikipedia.org/wiki/Bank_state_branch), without separators or whitespace.</value>
+        [DataMember(Name = "bsbCode", IsRequired = false, EmitDefaultValue = false)]
+        public string BsbCode { get; set; }
+
+        /// <summary>
+        /// Returns the string presentation of the object
+        /// </summary>
+        /// <returns>String presentation of the object</returns>
+        public override string ToString()
+        {
+            StringBuilder sb = new StringBuilder();
+            sb.Append("class AULocalAccountIdentification {\n");
+            sb.Append("  AccountNumber: ").Append(AccountNumber).Append("\n");
+            sb.Append("  BsbCode: ").Append(BsbCode).Append("\n");
+            sb.Append("  Type: ").Append(Type).Append("\n");
+            sb.Append("}\n");
+            return sb.ToString();
+        }
+
+        /// <summary>
+        /// Returns the JSON string presentation of the object
+        /// </summary>
+        /// <returns>JSON string presentation of the object</returns>
+        public virtual string ToJson()
+        {
+            return Newtonsoft.Json.JsonConvert.SerializeObject(this, Newtonsoft.Json.Formatting.Indented);
+        }
+
+        /// <summary>
+        /// Returns true if objects are equal
+        /// </summary>
+        /// <param name="input">Object to be compared</param>
+        /// <returns>Boolean</returns>
+        public override bool Equals(object input)
+        {
+            return this.Equals(input as AULocalAccountIdentification);
+        }
+
+        /// <summary>
+        /// Returns true if AULocalAccountIdentification instances are equal
+        /// </summary>
+        /// <param name="input">Instance of AULocalAccountIdentification to be compared</param>
+        /// <returns>Boolean</returns>
+        public bool Equals(AULocalAccountIdentification input)
+        {
+            if (input == null)
+            {
+                return false;
+            }
+            return 
+                (
+                    this.AccountNumber == input.AccountNumber ||
+                    (this.AccountNumber != null &&
+                    this.AccountNumber.Equals(input.AccountNumber))
+                ) && 
+                (
+                    this.BsbCode == input.BsbCode ||
+                    (this.BsbCode != null &&
+                    this.BsbCode.Equals(input.BsbCode))
+                ) && 
+                (
+                    this.Type == input.Type ||
+                    this.Type.Equals(input.Type)
+                );
+        }
+
+        /// <summary>
+        /// Gets the hash code
+        /// </summary>
+        /// <returns>Hash code</returns>
+        public override int GetHashCode()
+        {
+            unchecked // Overflow is fine, just wrap
```

Comment:
Remove comment: `// Overflow is fine, just wrap` (otherwise all the models will have the same comment) 👍

Reply:
Noted!